src/check-for-spam.js

```diff
@@ -0,0 +1,41 @@
+// content refers to the contents of the blog post (article) sent by feedparser
+module.exports = function (content) {
+  // Minimum words set to 20, can be changed
+  const MINWORDS = 20;
+
+  // Functions
+  function removeTags(str) { // Removes HTML tags from a string, leaving only plaintext
+    if (str != null && str !== '') {
+      return str.toString().replace(/<[^>]*>/g, '');
+    }
+    return '';
+  }
+
+  function getWordCount(str) { // Counts number of words in string
```

Comment:
We have another PR doing word count, see #17, #78.  You'll want to consume their data here.

Reply:
That feature appears to have issues and hasn't been merged yet so I left my own word counter in for now. I can implement it once it is finished.